### PR TITLE
Add minutes from today's meeting

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: docs/minutes/2019-01-08.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+- (jakirkham) use linter bot to update bot's graph with PR status
+
+#### ARM
+
+#### POWER
+
+#### CUDA
+
+* (jakirkham) CUDA 10.2 migration
+    * (CJ) the bot doesn't handle run_exports at all. It's particularly glaring with run_exports strong. Usually fuse host / run to figure dependencies. Does not use build for dependency tracking. Has some notes / issues that descirbes this a bit https://github.com/regro/cf-scripts/issues/677. TODO: Make a lookup table for the strong run_exports. Before making edges in graph, add in the run_exports strong requirements to the dependencies. This should propery express dependencies in the graph.
```
Comment (CJ-Wright) [inline docs/minutes/2019-01-08.md]:
```suggestion
    * (CJ) the bot doesn't handle run_exports at all. It's particularly glaring with run_exports strong. Usually fuse host / run to figure dependencies. Does not use build for dependency tracking. Has some notes / issues that describes this a bit https://github.com/regro/cf-scripts/issues/677. TODO: Make a lookup table for the strong run_exports. Before making edges in graph, add in the run_exports strong requirements to the dependencies. This should properly express dependencies in the graph.
```

**Comment by jakirkham** [comment]
@ericdill, could you please review the changes here when you have a chance? 🙂

**Comment by scopatz** [comment]
Let's just fix on master

**Comment by jakirkham** [comment]
@CJ-Wright, could you please send a PR?

**Comment by CJ-Wright** [comment]
Not today most likely

**Comment by jakirkham** [comment]
No worries. When you get a chance 😉

**Pull request merged**